draft-ecahc-moderation.md

```diff
@@ -247,28 +247,25 @@ moderator team in ways that are, ideally, integrated into the various
 participation channels the IETF offers. The moderator team SHALL keep
 the identities of participants requesting moderation confidential.
 
-## Transparency
-
-The moderator team SHALL operate according to a consistent
-set of criteria, processes, and actions. The moderator team SHALL
-independently define and execute their role. They SHALL maintain a
-public set of moderation criteria, processes, actions, and other
-material that allows the community to understand and comment on the
-role of the team. The moderator team SHOULD consider adopting
-moderation criteria, processes, and actions that other technical
-communities have found suitable. The moderator team's criteria and
-processes SHALL be developed with community input, but they are not
-expected to be documented in the RFC series.
-
-Some of these processes may rely on automated mechanisms, such as
-rate-limiting emails to lists or messages to chat channels.
-(The IETF's deliberately low bar to participation makes it easy to
-create throw-away personas for such denial-of-service behavior.)
+## Operations of the Moderator Team and Transparency
 
-{:aside}
-> TODO: This gives the moderator team broad freedom to define
-  processes and actions. Should this document define some boundaries
-  for what the moderator team can do?
+Within the bounds of the policies set within this memo and with the
+approval of the IESG, the moderator team SHALL define any additional
```

Comment:
This introduces the IESG into the moderation loop, which some have objected against.

Reply:
I don't think anyone but you and Keith have objected to the IESG having oversight.

Reply:
Excuse me: Dan as well.

Reply:
The current moderator team (for ietf-discuss) defines its policies and processes **without** the IESG needing to approve them. Why change this? Isn't the appeals path sufficient here? Introducing the IESG into any loop causes delays, always.

Reply:
That is **exactly** how the current moderator team operates.

Reply:
I know, and I'm saying, it's wrong.  This doesn't have to be complex, Lars.  The rules can be fairly broad.  But they don't just get to be their own little government, detached from everyone else, who can fall back on “But we followed the rules” because they set the rules.

Reply:
Maybe try "advice and consent of the IESG", which doesn't require formal approval but makes it clear that the mod team doesn't have _carte blanche_. Given that the rules will be published and there is an appeal path, I think that would be enough.

Reply:
@becarpenter I don't mind so long as it is clear that the moderators are themselves overseen.  This is a major difference between the two drafts that we started with, so I will bring it to the list

Reply:
OK. Before I merge this, should we as part of this PR change references to the IETF Chair to the IESG in the remainder of the doc?

Reply:
On this one, Lars I'm easy (enjoy that one while it lasts ;-).  Whatever you prefer.

Reply:
I think it would be more consistent to do so.

Reply:
Ok.  Do you want to just edit the PR and then push it?

Reply:
Done. Please take a look before I merge.

Reply:
LGTM